specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2022-11-01-preview/appplatform.json

```diff
@@ -8613,6 +8611,29 @@
         }
       }
     },
+    "DevToolPortalComponent": {
```

Comment:
>DevToolPortalComponent [](http://example.com/codeflow?start=5&length=22)

Do you intend to update the existing 2022-11-01-preview as well as the new 2023-01-01-preview api version?
Are customers going to be impacted by this change?

Reply:
Never mind, I see you changed these in a later commit.